cmd/api-gateway/main.go

```
@@ -37,12 +37,15 @@ const (
 )
 
 func main() {
-	if err := godotenv.Load(); err != nil {
-		slog.Info(logNoEnvFile)
-	}
+	_ = godotenv.Load() // Ignore error if .env file doesn't exist
```

Please don't ignore errors, at least log them